components/organisms/o-order-confirmation.vue

```diff
@@ -230,15 +230,15 @@ export default {
   }
 }
 .paragraph {
-  line-height: 1.875rem;
+  line-height: 1.8;
```

Comment:
why did you choose to change this?

Reply:
I was trying to convert line height to number format.